lib/components/table.vue

```diff
@@ -524,14 +533,25 @@
 
                 if (data.then && typeof data.then === 'function') {
                     // Provider returned Promise
-                    data.then((items) => {
+                    data.then(items => {
                         this._providerSetLocal(items);
                     });
                 } else {
                     // Provider returned Array data
                     this._providerSetLocal(data);
                 }
+            },
+            hasFormatter(field) {
+                return field.formatter && ((typeof (field.formatter) === 'function') || (typeof (field.formatter) === 'string'));
```

Comment:
Ah, so you are allowing each field to have it's own formatter function (via the fields objects)

I originally thought you were going to have a global field Formatter function (a prop), which would return a format based on the key, item, and field passed to it.  

Your solution allows for more flexibility. :)